docroot/modules/custom/mass_fields/mass_fields.module

```diff
@@ -694,7 +694,9 @@ function mass_fields_paragraph_view(array &$build, EntityInterface $entity, Enti
       $item['#options']['manual_description'] = $build['field_listitemdoc_desc_manual'];
       $item['#field_name'] = 'extra_list_manual_desc';
       // Vary the cache by the SHA of the description so that updating the description updates the display.
-      $item['#cache']['keys'][] = sha1($entity->field_listitemdoc_desc_manual->value);
+      if (!$entity->get('field_listitemdoc_desc_manual')->isEmpty()) {
```

Comment:
Lets please refactor $entity->field_listitemdoc_desc_manual into a bundle class and edit at least this file so it uses the new method.

Reply:
Done.